### ISSUE TITLE
Read previous notes and make a plan for updating the certificate

### ISSUE THREAD

**Issue by alanlawrence** [alanlawrence/helloweb#13]
The most useful hints were found [here](https://docs.google.com/document/d/1c9_Y9IWL_WkiGlD1HDiPBe7Sn7OtS1h-BPHpjyg2wjs/edit#heading=h.p19p08g8xjyj). Specifically:

1. Base64 encode the downloaded and joined alanjl_info.pem file
1. Paste it into the tls.crt field
1. Spin up the cluster and see what happens.

If all is good, then upload the modified ingress file to github. Since I need gitsecret to do that, it might not be possible from Cloud Console or my Docker container. So I will probably have to download it to DELL-XPS-8930, encrypt it and push it back to the repository from there. Then go back, back up the ingresss file, sync files in my Docker container and re-copy the plain backed up ingress file over the encrypted version that was created from it earlier.

Creating a [playbook here](https://docs.google.com/document/d/1Il3yMQeBXwcd5_9zN1A9EBFUy0QGXVwtYWxDwCbnsjk/edit#) and building it up as I work through all the challenges.

**Comment by alanlawrence** [comment]
Parent: #12

**Comment by alanlawrence** [comment]
Hit lots of challenges. The ingress spec became invalid as the api version I was using had been deprecated then removed. The new api version required a different layout. Then I downloaded the newly issude Rapid SSL online cert before vaidating my domain so I think I got the old one without realising which took me a long while to debug. Then curl was failing on my website despite everything working fine in browsers. The fix is to provide the full certificate chain in the ingress controller (one for another time).
